src/prpy/planning/ompl.py

```diff
@@ -192,38 +221,34 @@ def _SetPlannerRange(self, robot, ompl_args=None):
             # RRT-Connect's range to the same value used for collision
             # detection inside OpenRAVE.
             longest_extent = numpy.max(dof_ranges)
-            ompl_range = conservative_ratio * longest_extent - epsilon
+            ompl_range  = conservative_ratio * longest_extent - self.epsilon
+            ompl_range *= self.multiplier
 
             ompl_args['range'] = ompl_range
```

Comment:
Also, I think this `range` parameter is interpreted as a fraction, not as a resolution (in radians).